code/go/0chain.net/core/common/rate_limter.go

```diff
@@ -47,7 +48,11 @@ func UserRateLimit(handler ReqRespHandlerf) ReqRespHandlerf {
 		return Recover(handler)
 	}
 	return func(writer http.ResponseWriter, request *http.Request) {
-		tollbooth.LimitFuncHandler(userRateLimit.Limiter, Recover(handler)).ServeHTTP(writer, request)
+		if strings.HasPrefix(request.URL.Path, "/_") {
```

Comment:
is it for `_diagnostics` to bypass limiter?

Reply:
Yes, was for bypassing the diagnostic page for debugging, can be removed.